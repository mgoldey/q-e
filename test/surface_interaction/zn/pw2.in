&CONTROL
  calculation  = "scf",
  prefix       = "left",
  outdir       = "./out",
  pseudo_dir   = "../../pseudo",
  verbosity = "high" 
  wf_collect = .true.
/
&SYSTEM
  nosym = .true.
  nbnd = 14
  ibrav = 1,
  a = 12,
  nat = 2,
  ntyp = 1,
  ecutwfc = 50,
  tot_charge = +1
  nspin = 2,
  tot_magnetization = 1
/
&ELECTRONS
 ! startingpot = 'file'
 ! startingwfc = 'file'
  conv_thr    = 1.D-7,
  mixing_beta = 0.30,
/
EPCDFT
1 1.D+2 1.D-2 100 .true.
delta_charge 1 1 2 2 0.0 0.0

K_POINTS {Gamma}

ATOMIC_SPECIES
Zn 0.0 Zn.pbe-mt_fhi.UPF
ATOMIC_POSITIONS (angstrom)
Zn 5 5 4
Zn 5 5 2
