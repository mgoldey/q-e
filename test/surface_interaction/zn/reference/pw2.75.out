
     Program PWSCF v.5.1.1 starts on 10May2016 at 14:24: 6 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     4 processors
     R & G space division:  proc/nbgrp/npool/nimage =       4
     Waiting for input...
     Reading input from standard input
 Reading EPCDFT CARD. Printing based upon belief that input should be reproducable from output.
     Reading   1 constraint(s); tolerance:  100.000000 max change in field 0.01
     epcdft_update_intrvl: 100
     epcdft with surface: T
          constraint           type   A   A   D   D      T      L
                   1  delta_charge    1   1   2   2 0.0000 0.0000

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
               file Zn.pbe-mt_fhi.UPF: wavefunction(s)  4p 4f renormalized

     gamma-point specific algorithms are used
     Message from routine setup:
     the system is metallic, specify occupations

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used


     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        2047    2047    512               139165   139165   17446
     Max        2052    2052    514               139168   139168   17450
     Sum        8197    8197   2053               556667   556667   69791
     Tot        4099    4099   1027

     Generating pointlists ...
     new r_m :   0.0687 (alat units)  1.5590 (a.u.) for type    1


     bravais-lattice index     =            1
     lattice parameter (alat)  =      22.6767  a.u.
     unit-cell volume          =   11661.1222 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            1
     number of electrons       =        23.00 (up:  12.00, down:  11.00)
     number of Kohn-Sham states=           14
     kinetic-energy cutoff     =      50.0000  Ry
     charge density cutoff     =     200.0000  Ry
     convergence threshold     =      1.0E-07
     mixing beta               =       0.3000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)

     celldm(1)=  22.676714  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  1.000000 )  


     PseudoPot. # 1 for Zn read from file:
     ../../pseudo/Zn.pbe-mt_fhi.UPF
     MD5 check sum: 5596c9f2b89b9f7367257e7ea958b26e
     Pseudo is Norm-conserving, Zval = 12.0
     Generated using FHI98PP, converted with fhi2upf.x v.5.0.2
     Using radial grid of  527 points,  3 beta functions with: 
                l(1) =   1
                l(2) =   2
                l(3) =   3

     atomic species   valence    mass     pseudopotential
        Zn            12.00    65.39000     Zn( 1.00)

     Starting magnetic structure 
     atomic species   magnetization
        Zn           0.000

     No symmetry found


                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )
                  (     0          1          0      )
                  (     0          0          1      )

 cart.    s( 1) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


     point group C_1 (1)    
     there are  1 classes
     the character table:

       E    
A      1.00

     the symmetry operations in each class:
     E        1

   Cartesian axes

     site n.     atom                  positions (alat units)
         1           Zn  tau(   1) = (   0.4166667   0.4166667   0.3958333  )
         2           Zn  tau(   2) = (   0.4166667   0.4166667   0.2291667  )

   Crystallographic axes

     site n.     atom                  positions (cryst. coord.)
         1           Zn  tau(   1) = (  0.4166667  0.4166667  0.3958333  )
         2           Zn  tau(   2) = (  0.4166667  0.4166667  0.2291667  )

     number of k points=     2
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000
        k(    2) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000

                       cryst. coord.
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000
        k(    2) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000

     Dense  grid:   278334 G-vectors     FFT dimensions: ( 108, 108, 108)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         1.86 Mb     (    8725,   14)
        NL pseudopotentials             3.99 Mb     (    8725,   30)
        Each V/rho on FFT grid          9.61 Mb     (  314928,   2)
        Each G-vector array             0.53 Mb     (   69584)
        G-vector shells                 0.02 Mb     (    2154)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         3.73 Mb     (    8725,   56)
        Each subspace H/S matrix        0.02 Mb     (      56,   56)
        Each <psi_i|beta_j> matrix      0.00 Mb     (      30,   14)
        Arrays for rho mixing          38.44 Mb     (  314928,    8)

     Initial potential from superposition of free atoms
     Check: negative starting charge=(component1):   -0.041632
     Check: negative starting charge=(component2):   -0.041632

     starting charge   23.99975, renormalised to   23.00000

     negative rho (up, down):  3.990E-02 3.990E-02
     
     Adding image charge field due to neutral metal slab at origin in XY plane.
     Including monopole and dipole terms.
     Quadrupole not implemented yet.
     System should not overlap with cell edges.
     

     reference position (x0):         9.44863066    9.44863066    7.08647300 bohr
     Total charge   1.0000
     Total dipole moment   0.0004   0.0004   0.0001 au (Ha),   0.0010   0.0010   0.0003 Debye


     First surface image interaction energy:-0.144E+00[Ry]


     reference position (x0):         9.44863066    9.44863066    7.08647300 bohr
     Total charge   1.0000
     Total dipole moment   0.0004   0.0004   0.0001 au (Ha),   0.0010   0.0010   0.0003 Debye

     Starting wfc are   32 randomized atomic wfcs

     total cpu time spent up to now is        3.5 secs

     per-process dynamical memory:    78.4 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.00E-02,  avg # of iterations =  4.0

     negative rho (up, down):  2.722E-02 2.748E-02

     Magnetic moment per site:
     atom:    1    charge:    9.3445    magn:    0.0394    constr:    0.0000
     atom:    2    charge:    9.3314    magn:    0.0369    constr:    0.0000

     total cpu time spent up to now is        6.8 secs

     total energy              =    -217.97081875 Ry
     Harris-Foulkes estimate   =    -218.17614937 Ry
     estimated scf accuracy    <       0.54517004 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.00 Bohr mag/cell

     iteration #  2     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  2.37E-03,  avg # of iterations =  1.5

     negative rho (up, down):  1.241E-02 1.279E-02

     Magnetic moment per site:
     atom:    1    charge:    9.4658    magn:    0.0777    constr:    0.0000
     atom:    2    charge:    9.4540    magn:    0.0757    constr:    0.0000

     total cpu time spent up to now is        9.5 secs

     total energy              =    -217.96403870 Ry
     Harris-Foulkes estimate   =    -218.01288291 Ry
     estimated scf accuracy    <       0.11837476 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.01 Bohr mag/cell

     iteration #  3     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  5.15E-04,  avg # of iterations =  2.0

     negative rho (up, down):  6.529E-03 6.875E-03

     Magnetic moment per site:
     atom:    1    charge:    9.4588    magn:    0.0859    constr:    0.0000
     atom:    2    charge:    9.4546    magn:    0.0852    constr:    0.0000

     total cpu time spent up to now is       13.1 secs

     total energy              =    -217.99169283 Ry
     Harris-Foulkes estimate   =    -217.99028789 Ry
     estimated scf accuracy    <       0.00972164 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.04 Bohr mag/cell

     iteration #  4     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  4.23E-05,  avg # of iterations =  2.0

     negative rho (up, down):  2.830E-05 2.933E-05

     Magnetic moment per site:
     atom:    1    charge:    9.4778    magn:    0.0977    constr:    0.0000
     atom:    2    charge:    9.4749    magn:    0.0986    constr:    0.0000

     total cpu time spent up to now is       17.1 secs

     total energy              =    -217.99269877 Ry
     Harris-Foulkes estimate   =    -217.99260353 Ry
     estimated scf accuracy    <       0.00275592 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.05 Bohr mag/cell

     iteration #  5     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.20E-05,  avg # of iterations =  3.0

     negative rho (up, down):  2.141E-05 2.213E-05

     Magnetic moment per site:
     atom:    1    charge:    9.4712    magn:    0.0954    constr:    0.0000
     atom:    2    charge:    9.4681    magn:    0.0964    constr:    0.0000

     total cpu time spent up to now is       20.9 secs

     total energy              =    -217.99354783 Ry
     Harris-Foulkes estimate   =    -217.99342812 Ry
     estimated scf accuracy    <       0.00040713 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.08 Bohr mag/cell

     iteration #  6     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.77E-06,  avg # of iterations =  5.5

     Magnetic moment per site:
     atom:    1    charge:    9.4679    magn:    0.0932    constr:    0.0000
     atom:    2    charge:    9.4646    magn:    0.0943    constr:    0.0000

     total cpu time spent up to now is       24.5 secs

     total energy              =    -217.99351436 Ry
     Harris-Foulkes estimate   =    -217.99339225 Ry
     estimated scf accuracy    <       0.00003480 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.08 Bohr mag/cell

     iteration #  7     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  1.51E-07,  avg # of iterations =  3.0

     Magnetic moment per site:
     atom:    1    charge:    9.4676    magn:    0.0926    constr:    0.0000
     atom:    2    charge:    9.4644    magn:    0.0936    constr:    0.0000

     total cpu time spent up to now is       27.8 secs

     total energy              =    -217.99349419 Ry
     Harris-Foulkes estimate   =    -217.99347908 Ry
     estimated scf accuracy    <       0.00000604 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration #  8     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  2.63E-08,  avg # of iterations =  1.5

     Magnetic moment per site:
     atom:    1    charge:    9.4679    magn:    0.0924    constr:    0.0000
     atom:    2    charge:    9.4647    magn:    0.0934    constr:    0.0000

     total cpu time spent up to now is       30.7 secs

     total energy              =    -217.99347449 Ry
     Harris-Foulkes estimate   =    -217.99349863 Ry
     estimated scf accuracy    <       0.00000229 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration #  9     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  9.97E-09,  avg # of iterations =  2.5

     Magnetic moment per site:
     atom:    1    charge:    9.4679    magn:    0.0923    constr:    0.0000
     atom:    2    charge:    9.4648    magn:    0.0933    constr:    0.0000

     total cpu time spent up to now is       33.9 secs

     total energy              =    -217.99348959 Ry
     Harris-Foulkes estimate   =    -217.99349356 Ry
     estimated scf accuracy    <       0.00000022 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration # 10     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  9.45E-10,  avg # of iterations =  2.5

     reference position (x0):         9.44863066    9.44863066    7.08647300 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.3928 au (Ha),  -0.0010  -0.0010  -0.9983 Debye

       I         D           A         Val      Target            Str
       1 0.111E+01   0.101E+01   0.952E-01   0.000E+00   0.000000E+00
     Image interaction energy: -0.145E+00 [Ry] Change: -0.336E-03 [Ry] Tolerance:  0.100E-04 [Ry]

     Magnetic moment per site:
     atom:    1    charge:    9.4679    magn:    0.0923    constr:    0.0000
     atom:    2    charge:    9.4648    magn:    0.0933    constr:    0.0000

     total cpu time spent up to now is       37.5 secs

     total energy              =    -217.99349091 Ry
     Harris-Foulkes estimate   =    -217.99349414 Ry
     estimated scf accuracy    <       0.00000005 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     iteration # 11     ecut=    50.00 Ry     beta=0.30
     Davidson diagonalization with overlap
     ethr =  2.31E-10,  avg # of iterations =  1.0

     reference position (x0):         9.44863066    9.44863066    7.08647300 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.3928 au (Ha),  -0.0010  -0.0010  -0.9983 Debye

       I         D           A         Val      Target            Str
       1 0.111E+01   0.101E+01   0.952E-01   0.000E+00   0.000000E+00
     Image interaction energy: -0.145E+00 [Ry] Change:  0.129E-07 [Ry] Tolerance:  0.100E-04 [Ry]

     reference position (x0):         9.44863066    9.44863066    7.08647300 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.3928 au (Ha),  -0.0010  -0.0010  -0.9983 Debye


     Magnetic moment per site:
     atom:    1    charge:    9.4680    magn:    0.0923    constr:    0.0000
     atom:    2    charge:    9.4647    magn:    0.0933    constr:    0.0000

     total cpu time spent up to now is       40.2 secs

     End of self-consistent calculation

 ------ SPIN UP ------------


          k = 0.0000 0.0000 0.0000 ( 34896 PWs)   bands (ev):

   -13.5326   -12.8218   -12.8218   -12.0543   -12.0502   -11.8186   -11.8019   -11.5858
   -11.5858   -11.5396    -9.5975    -5.7154    -2.9823    -2.0765
     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   1.0000   0.0000   0.0000

 ------ SPIN DOWN ----------


          k = 0.0000 0.0000 0.0000 ( 34896 PWs)   bands (ev):

   -13.3993   -12.7121   -12.7121   -11.9771   -11.9730   -11.7389   -11.7223   -11.4314
   -11.4314   -11.3524    -9.3520    -4.7921    -2.6292    -2.6292
     occupation numbers 
     1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000   1.0000
     1.0000   1.0000   1.0000   0.0000   0.0000   0.0000

     highest occupied level (ev):    -5.7154

     reference position (x0):         9.44863066    9.44863066    7.08647300 bohr
     Total charge   1.0000
     Total dipole moment  -0.0004  -0.0004  -0.3928 au (Ha),  -0.0010  -0.0010  -0.9983 Debye

       I         D           A         Val      Target            Str
       1 0.111E+01   0.101E+01   0.952E-01   0.000E+00   0.000000E+00
     Image interaction energy: -0.145E+00 [Ry] Change:  0.129E-07 [Ry] Tolerance:  0.100E-04 [Ry]

!    total energy              =    -217.99347179 Ry
     Harris-Foulkes estimate   =    -217.99349414 Ry
     estimated scf accuracy    <       0.00000005 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =    -400.00554306 Ry
     hartree contribution      =     209.59536460 Ry
     xc contribution           =     -32.48655773 Ry
     ewald contribution        =       4.90223735 Ry
     CDFT correction           =      -0.00000000 Ry
     scf correction            =       0.00102705 Ry

     total magnetization       =     1.00 Bohr mag/cell
     absolute magnetization    =     1.09 Bohr mag/cell

     convergence has been achieved in  11 iterations

     Writing output data file left.save

     init_run     :      3.43s CPU      3.48s WALL (       1 calls)
     electrons    :     35.54s CPU     37.37s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      1.19s CPU      1.19s WALL (       1 calls)
     potinit      :      1.64s CPU      1.67s WALL (       1 calls)

     Called by electrons:
     c_bands      :     16.42s CPU     17.03s WALL (      11 calls)
     sum_band     :      4.93s CPU      5.50s WALL (      11 calls)
     v_of_rho     :     11.08s CPU     11.51s WALL (      11 calls)
     v_h          :      0.38s CPU      0.38s WALL (      11 calls)
     v_xc         :     10.70s CPU     11.13s WALL (      11 calls)
     mix_rho      :      1.55s CPU      1.58s WALL (      11 calls)

     Called by c_bands:
     init_us_2    :      0.35s CPU      0.36s WALL (      46 calls)
     regterg      :     15.69s CPU     16.29s WALL (      22 calls)

     Called by sum_band:

     Called by *egterg:
     h_psi        :     15.35s CPU     15.88s WALL (      81 calls)
     g_psi        :      0.06s CPU      0.06s WALL (      57 calls)
     rdiaghg      :      0.05s CPU      0.05s WALL (      79 calls)
     regterg:over :      0.49s CPU      0.53s WALL (      57 calls)
     regterg:upda :      0.29s CPU      0.31s WALL (      57 calls)
     regterg:last :      0.12s CPU      0.12s WALL (      22 calls)

     Called by h_psi:
     h_psi:vloc   :     14.50s CPU     15.02s WALL (      81 calls)
     h_psi:vnl    :      0.78s CPU      0.79s WALL (      81 calls)
     add_vuspsi   :      0.32s CPU      0.33s WALL (      81 calls)

     General routines
     calbec       :      0.46s CPU      0.46s WALL (      81 calls)
     fft          :      7.31s CPU      7.81s WALL (     292 calls)
     fftw         :     14.94s CPU     15.54s WALL (    1034 calls)

     Parallel routines
     fft_scatter  :     12.10s CPU     12.72s WALL (    1326 calls)

     PWSCF        :    39.17s CPU        41.10s WALL


   This run was terminated on:  14:24:47  10May2016            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
